doc_requirements.txt

```diff
@@ -2,5 +2,6 @@ mkdocs~=1.5.3
 mkdocstrings-python~=1.9.0
 black==24.3.0
 pygments~=2.17.2
+pulp-docs @ git+https://github.com/pedro-psb/pulp-docs@main
```

Comment:
Can we move such an important asset into the pulp area?

Reply:
Yes, as I said, one of the next goals of the working group is to move pulp-docs to pulp.
I don't think there is any particular reason stopping us from doing that right now, we just didn't have a chance to specifically discuss that yet.

Reply:
You did decided to use it, right? So getting it in the official namespace is not in need of further discussion. Just ask someone with permissions to move it.

Reply:
Will do